src/algebra/group/defs.lean

```diff
@@ -285,7 +285,7 @@ class add_cancel_comm_monoid (M : Type u)
   extends add_left_cancel_comm_monoid M, add_right_cancel_comm_monoid M
 
 /-- Commutative version of cancel_monoid. -/
-@[protect_proj, ancestor right_cancel_comm_monoid left_cancel_comm_monoid, to_additive add_cancel_comm_monoid]
+@[protect_proj, ancestor left_cancel_comm_monoid right_cancel_comm_monoid, to_additive add_cancel_comm_monoid]
```

Comment:
Turns out that by getting this backwards, `to_additive` would map `to_left_cancel_comm_monoid` to `to_add_right_cancel_comm_monoid`! The order matters.

Reply:
Perhaps we should add a warning about this to the [`to_additive` doc entry](https://github.com/leanprover-community/mathlib/blob/a1aa5110df112333ed27dc56f278328386642ed1/src/algebra/group/to_additive.lean#L174)?

Reply:
Sounds like a job for #5338

Reply:
Yes, fair enough.

Thanks!
bors r+